integration/caching_test.go

```diff
@@ -140,25 +156,25 @@ func testCaching(t *testing.T, context spec.G, it spec.S) {
 			Expect(firstImage.Buildpacks[1].Layers).To(HaveKey("launch-modules"))
 
 			Expect(logs).To(ContainLines(
-				fmt.Sprintf("%s 1.2.3", settings.Buildpack.Name),
-				"  Resolving installation process",
-				"    Process inputs:",
-				"      node_modules      -> \"Not found\"",
-				"      npm-cache         -> \"Not found\"",
-				"      package-lock.json -> \"Found\"",
-				"",
-				"    Selected NPM build process: 'npm ci'"))
+				fmt.Sprintf("%s%s 1.2.3", extenderBuildStr, settings.Buildpack.Name),
+				extenderBuildStr+"  Resolving installation process",
+				extenderBuildStr+"    Process inputs:",
+				extenderBuildStr+"      node_modules      -> \"Not found\"",
+				extenderBuildStr+"      npm-cache         -> \"Not found\"",
+				extenderBuildStr+"      package-lock.json -> \"Found\"",
+				extenderBuildStr+"",
+				extenderBuildStr+"    Selected NPM build process: 'npm ci'"))
 			Expect(logs).To(ContainLines(
-				"  Executing launch environment install process",
-				fmt.Sprintf("    Running 'npm ci --unsafe-perm --cache /layers/%s/npm-cache'", strings.ReplaceAll(settings.Buildpack.ID, "/", "_")),
+				extenderBuildStr+"  Executing launch environment install process",
+				fmt.Sprintf(extenderBuildStr+"    Running 'npm ci --unsafe-perm --cache /layers/%s/npm-cache'", strings.ReplaceAll(settings.Buildpack.ID, "/", "_")),
 			))
-			Expect(logs).To(ContainLines(MatchRegexp(`      Completed in (\d+\.\d+|\d{3})`)))
+			Expect(logs).To(ContainLines(MatchRegexp(`\[extender \(build\)\]       Completed in (\d+\.\d+|\d{3})`)))
```

Comment:
I think this line is the culprit, it should look more like [this](https://github.com/paketo-buildpacks/npm-install/pull/579/files#diff-22b157d6d07052d616bef4dd70d0304a8161d1753ea5ea7abf55ce0072125221R93-R95).

Reply:
Yeap, thanks, fixed!